superset/assets/javascripts/modules/utils.js

```diff
@@ -203,3 +203,11 @@ export function initJQueryAjaxCSRF() {
     });
   }
 }
+
+export function getTextWidth(text, fontDetails = '12px Roboto') {
```

Comment:
yeah, agreed, maybe into a `visUtils` module since this is useful for measuring labels on an axis as well.

i don't know how expensive this is exactly, but since we are not actually drawing the element on to the screen i don't think it should be too expensive. definitely an area to investigate to improve performance.